src/test/java/edu/ucla/library/iiif/fester/CsvParserTest.java

```diff
@@ -45,6 +45,18 @@ public final void testParse() throws CsvParsingException, CsvException, IOExcept
         myCsvParser.parse(getTestPath(GOOD_CSV));
     }
 
+    /**
+     * Tests handling spaces in non-header rows in a CSV file.
+     *
+     * @throws CsvParsingException If there is an error while parsing the CSV data
+     * @throws CsvException If there is a generic CSV error
+     * @throws IOException If there is trouble reading the CSV data
+     */
+    @Test
+    public final void testParseSpaces() throws CsvParsingException, CsvException, IOException {
+        myCsvParser.parse(getTestPath("spaces.csv"));
```

Comment:
This tests that there were no exceptions thrown, but it would be good to test that the " Calif. Postcards Collection.  Collection 1351" string no longer has the space in front of it, too (I know it doesn't, but it's good to add that check to the test in case we ever break the functionality you've added). One way to do that would be to get `CsvMetadata metadata = myCsvParser.parse(getTestPath("spaces.csv")).getCsvMetadata()` and then check that the first string in the seventh string array (in the `getWorksList()` from `metadata`) starts with "C" instead of " ". Everything else looks good to me.